### PR TITLE
Fixed label translation in validation messages (Fixed Issue #130)

### PR DESCRIPTION
**Pull request by billmn**

### CODE REVIEW

**Comment by billmn** [comment]
Added fix for possible undefined index

File: classes/observer/validation.php [hunk 1]
```diff
@@ -96,7 +96,7 @@ public static function set_fields($obj, $fieldset = null)
 			$attributes = isset($settings['form']) ? $settings['form'] : array();
 			// label is either set in property setting, as part of form attributes or defaults to fieldname
 			$label = isset($settings['label']) ?
-				$settings['label'] : (isset($attributes['label']) ? $attributes['label'] : $p);
+				\Lang::get($settings['label'], array(), $settings['label']) : (isset($attributes['label']) ? \Lang::get($attributes['label'], array(), $attributes['label']) : $p);
```
Comment (jschreuder) [inline classes/observer/validation.php:100]: It's gotten a bit too long, it'd be better to devide this over 2 lines:

```
$label = isset($settings['label']) ? $settings['label'] : (isset($attributes['label']) ? $attributes['label'] : $p);
$label = \Lang::get($label) ?: $label;
```

Comment (billmn) [inline classes/observer/validation.php:100]: yeah ... is true .. I make the change and send a new pull-request

**Comment by billmn** [comment]
sent .... :)

**Pull request merged**